lib/opFns.js

```diff
@@ -766,28 +775,16 @@ function trap (err) {
 /**
  * Subtracts the amount needed for memory usage from `runState.gasLeft`
  * @method subMemUsage
- * @param {Number} offset
- * @param {Number} length
+ * @param {BN} offset
+ * @param {BN} length
  * @return {String}
  */
 function subMemUsage (runState, offset, length) {
-  //  abort if no usage
-  if (!length) return
```

Comment:
So apparently this one is the correct behaviour as per the sidenote in the YP in section H.1.